src/assets/api-calls.js

```diff
@@ -32,3 +32,24 @@ export function updateArticleVotes(article_id, count) {
       return data.article.votes;
     });
 }
+
+export function postComment(article_id, newComment) {
+  const body = {
```

Comment:
we pass in an object which already looks like this, do we need to then build up that object again?